src/legacy/ui/public/vis/draggable/draggable_handle.js

```diff
@@ -1,33 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import { uiModules } from '../../modules';
-
-uiModules
-  .get('kibana')
-  .directive('draggableHandle', function () {
-    return {
-      restrict: 'A',
-      require: '^draggableItem',
-      link($scope, $el, attr, ctrl) {
-        ctrl.registerHandle($el);
-        $el.addClass('gu-handle');
```

Comment:
Please, remove classes `gu-handle` and `gu-mirror` from `src\legacy\core_plugins\kibana\public\management\_hacks.scss` since they are unused

Reply:
Thanks. Nice catch!